multi-node/aws/pkg/config/templates/cloud-config-etcd

```diff
@@ -0,0 +1,122 @@
+#cloud-config
+coreos:
+  update:
+    reboot-strategy: etcd-lock
+  units:
+    - name: etcd2.service
+      drop-ins:
```

Comment:
Any specific reason to use a drop-in vs letting cloud-config do the translation? https://coreos.com/os/docs/latest/cloud-config.html#etcd2